index.html

```diff
@@ -4,255 +4,336 @@
-                        <h5>Multi-Post Stories Gain+Glory</h5>
-                        <ul class="card_languages">
-                            <li class="language"><p>Ruby on rails</p></li>
-                            <li class="language"><p>css</p></li>
-                            <li class="language"><p>JavaScript</p></li>
-                            <li class="language"><p>html</p></li>
-                        </ul>
-                        <button  type="button" class="buttons">See Project</button>
-                    </div>
+    <section class="home">
+      <div class="headlines">
+        <h1 class="headline-1">Hey there. I’m Souad</h1>
+        <h2 class="headline-2">I’m a software developer</h2>
+      </div>
+      <div class="headline-description">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
 
-                </div>
-                <div class="card">
-                    <div class="card_img"></div>
-                    <div class="card_info">
-                        <h5>Multi-Post Stories Gain+Glory</h5>
-                        <ul class="card_languages">
-                            <li class="language"><p>Ruby on rails</p></li>
-                            <li class="language"><p>css</p></li>
-                            <li class="language"><p>JavaScript</p></li>
-                            <li class="language"><p>html</p></li>
-                        </ul>
-                        <button type="button" class="buttons">See Project</button>
-                    </div>
+      <div class="social_media">
+        <ul class="links">
+          <li>
+            <a href="https://github.com/souad988"
+              ><img src="media/Icons/github.png" alt="github"
+            /></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/souadElmansouri"
+              ><img src="media/Icons/twitter.png" alt="twiter"
+            /></a>
+          </li>
+          <li>
+            <a href="www.linkedin.com/in/souad-el-mansouri-4725491a0"
+              ><img src="media/Icons/Linkedin.png" alt="linkdin"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.angellist.com/"
+              ><img src="media/Icons/angellist.svg" alt="angellist"
+            /></a>
+          </li>
+        </ul>
+      </div>
+      <div class="scrolldown">
+        <a href="#bottom"
+          ><img
+            src="media/Icons/NormalButton/Secondary/Icons/down.png"
+            alt="scroll_down"
+        /></a>
+      </div>
+    </section>
+    <section id="my-work" class="middle_section">
+      <div class="works_section_title">
+        <h3 class="works_headline">My Recent Works</h3>
+        <img
+          src="./media/images/indicator.png"
```

Comment:
- [ ] **Optional:** This image doesn't load, it'd be a good idea to get it working, don't you think?
![image](https://user-images.githubusercontent.com/27709832/150976567-ff955939-5388-443a-9878-6e85ab00c77e.png)

Reply:
thanks for reviewing, if you can help i dont know why it is working on my local repos but not github pages even if the url is the same